modules/docs/lib/widgetUtils.tsx

```diff
@@ -173,7 +173,7 @@ export const SymbolDialog = ({value}: SymbolDialogProps) => {
                   {breadcrumbsList.map((item, index) => {
                     return (
                       <>
-                        {item === breadcrumbsList[breadcrumbsList.length - 1] ? (
+                        {index === breadcrumbsList.length - 1 ? (
```

Comment:
This change fixes where a breadcrumb item is the same string as the previous.

For example:

> Some Item > Nested ItemNested ItemNestedItem

This fix corrects the breadcrumb to render like

> Some Item > Nested Item > Nested Item > Nested Item